src/HyeEun/components/common/ActionButton.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+import theme from "../../style";
+
+type ActionButtonProps = {
+  children: string;
+};
+
+export default function ActionButton({ children }: ActionButtonProps) {
+  return <St.ActionButton>{children}</St.ActionButton>;
+}
```

Comment:
버튼 태그에는 꼭 `type` 을 명시해주세요!!

default type 이 submit 이기 때문에, form 태그를 함께 사용한다면 페이지가 새로고침 될 수 있습니다!

```suggestion
export default function ActionButton({ children }: ActionButtonProps) {
  return <St.ActionButton type="button">{children}</St.ActionButton>;
}
```